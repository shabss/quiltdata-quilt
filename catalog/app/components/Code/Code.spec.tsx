import * as React from 'react'
import renderer from 'react-test-renderer'
import * as M from '@material-ui/core'

import * as style from 'constants/style'

import Code from './'

describe('components/Code', () => {
  it('should render', () => {
    const tree = renderer
      .create(
        <M.MuiThemeProvider theme={style.appTheme}>
          <Code className="B">A</Code>
        </M.MuiThemeProvider>,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
