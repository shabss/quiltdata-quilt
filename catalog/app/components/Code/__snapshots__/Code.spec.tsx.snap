// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Code should render 1`] = `
<code
  className="makeStyles-root-1 B"
>
  A
</code>
`;
